### PR TITLE
Prepare 7.1.1 release

### PR DESCRIPTION
**Pull request by deguif**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,16 @@ All notable changes to this project will be documented in this file.
...
+### Backward Compatibility Breaks
+### Added
+### Changed
+### Deprecated
+### Removed
+### Fixed
+### Security
+
+## [7.1.1](https://github.com/ruflin/Elastica/compare/7.1.0...7.1.1)
 ### Backward Compatibility Breaks
```
Comment (ruflin) [inline CHANGELOG.md:17]: Nit: You could get rid of the empty titles.

**Review by ruflin — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (deguif) [reply CHANGELOG.md:17]: Oups. Im fixing it.

**Review by ruflin — APPROVED** [review]

**Comment by ruflin** [comment]
Not sure if I ever linked this: https://elastica.io/release/how-to-create-an-elastica-release.html It is my release check list. You were already mostly following this. One thing we didn't do for the past few releases is updated elastica.io with the release and push new docs. But I think this is fine to reduce maintenance.

**Comment by deguif** [comment]
Regarding the `elastic.io` update, maybe we can automate it with manual github action?

**Review by thePanz — APPROVED** [review]

**Comment by ruflin** [comment]
@deguif Interesting idea! Github actions for the win. But I would not set a lot of priority on it at the moment as not sure how many people actually still check it out (I need to check the page stats ;-) ).

**Review by ruflin — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -31,12 +31,12 @@ Versions & Dependencies
...
-| [7.0](https://github.com/ruflin/Elastica/tree/7.0)                                      | 7.x           | ^7.0              | ^7.2         |
-| [6.x](https://github.com/ruflin/Elastica/tree/6.x)                                      | 6.x           | ^6.0              | ^7.0         |
-| [5.x](https://github.com/ruflin/Elastica/tree/5.x) (unmaintained)                       | 5.x           | ^5.0              | \>=5.6       |
-| [3.2.3](https://github.com/ruflin/Elastica/tree/3.2.3) (unmaintained)                   | 2.4.0         | no                | \>=5.4       |
-| [2.x](https://github.com/ruflin/Elastica/tree/2.x) (unmaintained)                       | 1.7.2         | no                | \>=5.3.3     |
+| Elastica                                                                                | ElasticSearch | elasticsearch-php | PHP            |
+| --------------------------------------------------------------------------------------- | ------------- | ----------------- | -------------- |
+| [7.x](https://github.com/ruflin/Elastica/tree/master)                                   | 7.x           | ^7.0              | ^7.2 \|\| ^8.0 |
```
Comment (deguif) [inline README.md:36]: Escape the pipe as it breaks the line.

**Pull request merged**